components/hooks/useMediaQuery.ts

```diff
@@ -4,26 +4,20 @@ const useMediaQuery = (width: number) => {
   const [targetReached, setTargetReached] = useState(false);
 
   const updateTarget = useCallback((e: MediaQueryListEvent) => {
```

Comment:
This is all fine and it does all make the code tidier. But please to keep PR's focused and concise to the issue. If you see things you want to refactor, please submit a issue request and we are happy to make that for you and you can pick it up.